### DISCUSSION TITLE
Update to German (only open fields)

### DISCUSSION THREAD

**Discussion by Rosenholz52** [aclap-dev/video-downloadhelper#958] [Language & Translation]
See the file attached.
[de_Rosenholz52.txt](https://github.com/user-attachments/files/16066457/de_Rosenholz52.txt)
Bye

**Comment by halkingb** [comment]
[de.txt](https://github.com/user-attachments/files/16072339/de.txt)

**Comment by EoSandwich** [comment]
[vdh-de_2024-07-04.txt](https://github.com/user-attachments/files/16098241/vdh-de_2024-07-04.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
